src/metrics.ts

```diff
@@ -97,7 +97,7 @@ export default class Metrics extends EventEmitter {
 
   private url: string;
 
-  private timer: NodeJS.Timer | undefined;
```

Comment:
how did you find that this needs to change?

Reply:
I had to update Node.js types to fix other error
```
../node_modules/@types/node/module.d.ts:121:13 - error TS2386: Overload signatures must all be optional or required.

121             resolve?(specified: string, parent?: string | URL): Promise<string>;
                ~~~~~~~
```
And new types where complaining about NodeJS.Timer

Reply:
BTW, we should add Node v22 to our test array.
https://nodejs.org/en/about/previous-releases